sites/svelte-5-preview/src/routes/docs/content/03-appendix/02-breaking-changes.md

```diff
@@ -203,3 +203,13 @@ Previously, bindings did not take into account `reset` event of forms, and there
 ### `walk` not longer exported
 
 `svelte/compiler` reexported `walk` from `estree-walker` for convenience. This is no longer true in Svelte 5, import it directly from that package instead in case you need it.
+
+### `<svelte:element>` tag must be an expression
+
+In Svelte 4, `<svelte:element this="div">` is valid code. This makes little sense — you should just do `<div>`. In the vanishingly rare case that you _do_ need to use a literal value for some reason, you can do this:
+
+```svelte
+<svelte:element this="div">...</svelte:element>
```

Comment:
```suggestion
```svelte
<svelte:element this={"div"}>...</svelte:element>
```

As this typo shows, the difference is subtle and may be hard to notice, so maybe a diff block would be even better:

```suggestion
```diff
- <svelte:element this="div">
+ <svelte:element this={"div"}>
```

Reply:
d'oh!